packages/config/config/devices/0x0159/zmnhtd.json

```
@@ -98,12 +98,12 @@
 		},
 		{
 			"#": "42",
-			"label": "Power reporting (Watts) by time interval",
-			"unit": "%",
+			"label": "Power reporting (Watts) by time interval (0-59 = reporting disabled)",
```

```suggestion
			"label": "Power (W) Reporting Interval",
			"description": "Allowable range: 60-32535",
```
and under the default value add an option:
```jsonc
			"options": [
				{
					"label": "Disable",
					"value": 0
				}
			]
```